docs/source/development.md

```diff
@@ -10,6 +10,18 @@ Information for contributors
 * Run type hint checks: `mypy -p forest`
 * Run test suite: `pytest`
 
+### Documentation
+Install required dependencies:
+```shell
+pip install docs/requirements.txt
+```
+Build the docs:
+```shell
+cd docs/source
```

Comment:
I believe this is just cd docs, or at least I wasn't able to use "make html" from docs/source